macros/src/lib.rs

```diff
@@ -449,9 +449,9 @@ fn log(level: MLevel, ts: TokenStream) -> TokenStream {
     let logging_enabled = is_logging_enabled(level);
     quote!({
         if #logging_enabled {
-            if let Some(mut _fmt_) = defmt::export::acquire() {
-                match (defmt::export::timestamp(), #(&(#args)),*) {
-                    (ts, #(#pats),*) => {
+            match (defmt::export::timestamp(), #(&(#args)),*) {
```

Comment:
let's evaluate `timestamp` last so that when you use the AtomicUsize timestamp function the timestamps will appear in order. I think right now something like this `info!("x={:?}", foo())` will print
``` text
0.000002 foo evaluated
0.000001 x=something
```

Reply:
Yeah, sounds good